aptos-move/block-executor/src/view.rs

```diff
@@ -568,7 +570,7 @@ impl<'a, T: Transaction> ParallelState<'a, T> {
                     unreachable!("Reading group size does not require a specific tag look-up");
                 },
                 Err(Dependency(dep_idx)) => {
-                    if !wait_for_dependency(self.scheduler, txn_idx, dep_idx)? {
+                    if !wait_for_dependency(&self.scheduler, txn_idx, dep_idx)? {
```

Comment:
why do we need this wrapper function? just calling self.scheduler.wait_for_dependency is not enough?